standalone/standalone.tpl

```diff
@@ -1478,6 +1469,31 @@ def create_eval_job(
         ),
     )
 
+    if judge_serving_model_ca_cert:
+        # Define the volume that references the Secret
+        secret_volume = kubernetes.client.V1Volume(
+            name="judge-ca-cert-volume",
+            secret=kubernetes.client.V1SecretVolumeSource(
```

Comment:
are we fetching the CA bundle from secret or configmap?

Reply:
Good point, forgot to update to use a CM :)